config/default.schema.json

```diff
@@ -245,7 +252,33 @@
 		"S3_KEY": {
 			"type": "string",
 			"description": "Synchronous encryption key used to secure storage provider secrets. Only used with FEATURE_MULTIPLE_S3_PROVIDERS_ENABLED=true",
-			"pattern": ".{16}.*"
+			"pattern": ".{16}.*",
+			"default": "abcdefabcdefabcdefa"
```

Comment:
```suggestion
			
```
there should not be a default being applied, this must be set in production - remove this line